qualification/report/generate_pdf.py

```diff
@@ -876,17 +894,21 @@ def main():
     parser.add_argument("input", help="Output of pytest --report-log=...")
     parser.add_argument("pdf", help="PDF file to write")
     parser.add_argument("-c", "--commit-id", action="store_true", help="Output commit ID of katgpucbf image")
+    parser.add_argument("--generate-procedure-doc", action="store_false", dest="make_report", default=True)
```

Comment:
What do you think about using `--doc=report` / `--doc=procedure` rather than a boolean flag (you could still have a default)? Sooner or later someone might want to generate something else. Similarly the argument to `document_from_list` could be a string ("report" or "procedure") rather than a boolean.

Reply:
I've given this some thought, and I think on balance I'd let the YAGNI principle be the guiding one here.

I don't forsee (right now) another kind of document that we might want to generate with this specific code. Handling sets of valid strings is also more tricky than simple booleans.

If the need arises, it's straightforward enough to add the functionality, but I don't see a need right now.

Reply:
This should probably have a help string too.